Updates images and documents new dashboards and filters (#3784)
elastic/security-docs: docs/dashboards/dashboards-overview.asciidoc

:frontmatter-tags-content-type: [overview]
:frontmatter-tags-user-goals: [visualize, monitor, analyze]

This section documents the {security-app}'s default dashboards, which provide visualizations of your security environment. To view them in {security-app}, select *Dashboards* from the navigation menu.
From the Dashboards page, you can access the default dashboards, as well as create and access custom dashboards.

> To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by using the text search field to filter by name and description, or by using the *Tags* menu to filter by tag. You can also click one of a custom dashboard's tags to toggle whether to filter for that tag. To create a new tag, open the *Tags* menu and click *Manage tags*.

Reviewer: ```suggestion
To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by:
* Using the text search field to filter by name or description
* Using the **Tags** menu to filter by tag
* Clicking one of a custom dashboard's tags to toggle whether to filter for that tag
To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
```
A couple of tiny suggestions, otherwise LGTM!

(The reason I suggested replacing single asterisks with double is to reduce the potential amount of [fixes after the migration to MDX](https://github.com/elastic/staging-serverless-security-docs/pull/49#discussion_r1297444042).)

:frontmatter-tags-content-type: [overview]
:frontmatter-tags-user-goals: [visualize, monitor, analyze]

This section documents the {security-app}'s default dashboards, which provide visualizations of your security environment. To view them in {security-app}, select *Dashboards* from the navigation menu.
From the Dashboards page, you can access the default dashboards, as well as create and access custom dashboards.

To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by:
* Using the text search field to filter by name or description
* Using the **Tags** menu to filter by tag
* Clicking one of a custom dashboard's tags to toggle whether to filter for that tag
To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.

image::images/dashboards-landing-page.png[The Dashboards landing page]

> For more information about each of the default dashboards, refer to the other pages in the Dashboards section. For more information about creating custom dashboards, refer to {kibana-ref}/create-a-dashboard-of-panels-with-web-server-data.html[Create your first Kibana dashboard].

Reviewer 2: Is the first sentence pointing users towards docs for the other dashboards? If yes, it might be more clear if you replace "pages" with "documentation". Here's an example:

```suggestion
Refer to documentation for the other {elastic-sec} dashboards to learn more about them. For more information about creating custom dashboards, refer to {kibana-ref}/create-a-dashboard-of-panels-with-web-server-data.html[Create your first Kibana dashboard].
```

:frontmatter-tags-content-type: [overview]
:frontmatter-tags-user-goals: [visualize, monitor, analyze]

> The {security-app}'s default dashboards provide useful visualizations of your security environment. To view them in {security-app}, select *Dashboards* from the navigation menu.

Reviewer 3: ```suggestion
The {security-app}'s default dashboards provide useful visualizations of your security environment. To view them in {elastic-sec}, select *Dashboards* from the navigation menu.
```

From the Dashboards page, you can access the default dashboards, as well as create and access custom dashboards.

> To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table by:
> 
> * Using the text search field to filter by name or description
> * Using the **Tags** menu to filter by tag
> * Clicking one of a custom dashboard's tags to toggle whether to filter for that tag
> To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.

Reviewer 3: A few fixes here. Partly subjective: basic imperative verbs (vs. gerunds) are more direct, especially for commands/suggestions, and they tend to encourage slower reading through the whole list (vs. quicker scanning). Also, need another blank line to end the list and start a new paragraph.
```suggestion
To create a new custom dashboard, click **Create Dashboard**. You can control which custom dashboards appear in the table:

* Use the text search field to filter by name or description.
* Use the **Tags** menu to filter by tag.
* Click a custom dashboard's tags to toggle filtering for each tag.

To create a new tag or edit existing tags, open the **Tags** menu and click **Manage tags**.
```
